### ISSUE TITLE
[compiler] DirectCallNodes not correctly reported with new language-agnostic inlining

### ISSUE THREAD

**Issue by fniephaus** [oracle/graal#2665]
**Describe the issue**
When running with the new inlining, the DirectCallNodes stats are all zero (`I    0/D    0`) and don't show the actual values. Inlining of direct calls is still working, but this is valuable debug information to understand the behavior of the new inliner.

**Steps to reproduce the issue**
Run with `-Dgraal.TruffleLanguageAgnosticInlining=true`.

**Describe GraalVM and your environment:**
 - GraalVM version: **CE 20.1.0**
 - JDK major version: **11**
 - OS: **macOS Catalina**
 - Architecture: **AMD64**

**More details**

```
[engine] opt done         Queens>>row:column: <opt>                                   |ASTSize     139/  139 |Time   372( 203+169 )ms |DirectCallNodes I    0/D    0 |GraalNodes   284/  383 |CodeSize         1660 |CodeAddress 0x1261ac010 |Source            n/a
[engine] opt done         Queens>>placeQueen: <opt> <split-288b5fe>                   |ASTSize     232/  232 |Time  3894(1408+2486)ms |DirectCallNodes I    0/D    0 |GraalNodes  2735/ 3568 |CodeSize        19558 |CodeAddress 0x1261c8c90 |Source            n/a
[engine] opt done         Queens>>placeQueen: <opt> <split-52b4d927>                  |ASTSize     231/  231 |Time  3924(1422+2501)ms |DirectCallNodes I    0/D    0 |GraalNodes  2726/ 3566 |CodeSize        22054 |CodeAddress 0x1261dfb90 |Source            n/a
[engine] opt done         Queens>>placeQueen: <opt>                                   |ASTSize     232/  232 |Time  3957(1369+2588)ms |DirectCallNodes I    0/D    0 |GraalNodes  2735/ 3568 |CodeSize        21114 |CodeAddress 0x1261f7590 |Source            n/a
[engine] opt done         Queens>>row:column:put: <opt>                               |ASTSize     148/  148 |Time   241( 116+125 )ms |DirectCallNodes I    0/D    0 |GraalNodes   246/  292 |CodeSize         1240 |CodeAddress 0x126207990 |Source            n/a
[engine] opt done         Queens>>placeQueen: <opt> <split-32e5f7ac>                  |ASTSize     231/  231 |Time  4239(1428+2811)ms |DirectCallNodes I    0/D    0 |GraalNodes  2726/ 3566 |CodeSize        21841 |CodeAddress 0x12620b390 |Source            n/a
[engine] opt done         SequenceableCollection>>atAllPut: <opt>                     |ASTSize     129/  129 |Time   521( 122+400 )ms |DirectCallNodes I    0/D    0 |GraalNodes   293/  705 |CodeSize         3515 |CodeAddress 0x126219490 |Source            n/a
[engine] opt done         Queens>>placeQueen: <opt> <split-7a5e8a59>                  |ASTSize     231/  231 |Time  4202(1471+2731)ms |DirectCallNodes I    0/D    0 |GraalNodes  2726/ 3566 |CodeSize        21318 |CodeAddress 0x126222c90 |Source            n/a
[engine] opt done         Queens>>placeQueen: <opt> <split-1d8bac3e>                  |ASTSize     230/  230 |Time  4220(1616+2604)ms |DirectCallNodes I    0/D    0 |GraalNodes  2698/ 3521 |CodeSize        20145 |CodeAddress 0x126236c90 |Source            n/a
[engine] opt done         Queens>>benchmark <opt>                                     |ASTSize     103/  103 |Time  5718(1838+3880)ms |DirectCallNodes I    0/D    0 |GraalNodes  3305/ 4280 |CodeSize        18694 |CodeAddress 0x12624da90 |Source            n/a
[engine] opt done         Queens>>verifyResult: <opt>                                 |ASTSize      11/   11 |Time    33(  25+8   )ms |DirectCallNodes I    0/D    0 |GraalNodes    21/   15 |CodeSize          140 |CodeAddress 0x12625ef10 |Source            n/a
```

Please assign to @boris-spas.

**Comment by boris-spas** [comment]
Thanks for the report... Already tracked internally as GR-24644 :)

**Comment by boris-spas** [comment]
Addressed in d6816740ee697b99c43e8d7112cd368a9557a965.
